src/ResourceManagement/RecoveryServices.Backup/RecoveryServices.Tests/ScenarioTests/AzureIaasVMTests/RestoreTests.cs

```diff
@@ -71,6 +71,14 @@ public void RestoreDiskTest()
                     TargetDomainNameId = string.Empty,
                     TargetResourceGroupId = string.Empty,
                     TargetVirtualMachineId = string.Empty,
+                    EncryptionDetails = new EncryptionDetails()
+                    {
+                        EncryptionEnabled = true,
+                        KekUrl = ConfigurationManager.AppSettings["KekUrl"],
```

Comment:
As discussed, these values should no longer be coming from app.config - these should be created as part of the test